llvm/test/CodeGen/PowerPC/subreg-coalescer.mir

```diff
@@ -0,0 +1,26 @@
+# RUN: llc -mtriple powerpc64le-unknown-linux-gnu -mcpu=pwr8 -x mir < %s \
+# RUN:   -verify-machineinstrs --run-pass=register-coalescer -o - | FileCheck %s
```

Comment:
-verify-coalescing would be more useful than -verify-machineinstrs. Also, should use update_mir_test_checks

Reply:
Also not using stdin and directly reading the file avoids the need for -x mir